test/vinyl/gh-5823-crash_snapshot.lua

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env tarantool
+
+--
+-- mode == 0: casual bootstrap;
+-- mode == 1: force recovery bootstrap;
+-- mode == 2: casual bootstrap and fill in data.
+--
+local mode = tonumber(arg[1])
+box.cfg ({
+    replication = os.getenv("MASTER"),
+    vinyl_memory = 128 * 1024 * 1024,
+    force_recovery = (mode == 1),
+})
+
+if mode == 2 then
+    local v = box.schema.space.create('test_v', {engine = 'vinyl'})
+    v:create_index('pk', {run_count_per_level = 100, page_size = 128, range_size = 1024})
```

Comment:
Why do you need so specific settings for the index?

Reply:
Actually, I don't need them. Removed